setup.py

```
@@ -0,0 +1,22 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='gitbi',
+    version='0.1.0',
+    packages=find_packages('gitbi'),
+    install_requires=[
+    ],
+    author='Reuven Peleg',
+    author_email='your.email@example.com',
+    description='A tool for automating git bisect operations',
+    long_description=open('README.md', 'r', encoding='utf-8').read() if os.path.exists('README.md') else '',
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)